app/components/Home.js

```diff
@@ -158,7 +164,7 @@ export default class Home extends Component {
               }
             </Description>
             <Divider padded={1.5} />
-            <CountDown endDate={new Date().getTime() + 100000000} />
+            <CountDown endDate={this.endDate(auctionOfInterest)} />
```

Comment:
This and `Auction.js` should use the `utils` one. And remove the duplicate code

Reply:
It does use the `utils` one. It just calls the local method which redirects to the utils method with the required inputs.

Reply:
Hmm, that seems kinda weird. This still duplicates the code (and actually leads to more code probably). Maybe `getEndDate` should accept an auction then?

Reply:
Don't necessarily agree. Computation is shared logic, but supplying data to the computation is different.  I changed anyway to do it a bit differently.